src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,110 @@
+package duke.parser;
+
+import duke.command.Command;
+import duke.command.ByeCommand;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.ListCommand;
+import duke.command.TodoCommand;
+import duke.command.FindCommand;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.UnknownCommandException;
+import duke.exceptions.WrongSyntaxException;
+import duke.exceptions.EmptyBodyException;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
```

Comment:
I like the splitting of import chunks, very readable. Shall do that for mine too